src/battle_util.c

```diff
@@ -8964,7 +8964,11 @@ static inline u32 CalcMoveBasePowerAfterModifiers(u32 move, u32 battlerAtk, u32
         break;
     case ABILITY_TRANSISTOR:
         if (moveType == TYPE_ELECTRIC)
+        #if B_TRANSISTOR >= GEN_9
+            modifier = uq4_12_multiply(modifier, UQ_4_12(5325 / 4096));
```

Comment:
Damn, this isn't quite `UQ_4_12(1.3)`—`1.3 * 4096` is `5324.8` which truncates down to `5324`. I wonder if we could change it to round :thinking:

Anyway, I won't ask you to do that! Just something for me to look at for a future PR :)

Reply:
I did try 1.3 while having the actual division in the code, but the test failed so having it here was the only way to make it pass :( it's unfortunate indeed, but we could always just take artistic license and simplify it to 1.3

Reply:
I want to see where this goes before accepting it.